addons/website_blog/static/src/snippets/s_blog_posts/000.scss

```diff
@@ -1,64 +1,42 @@
 .s_dynamic {
-    .s_blog_posts_post_title  {
-        font-weight: $headings-font-weight;
-        // Tweek line-height to help fit multi-line titles.
-        line-height: 1;
-    }
-
-    .s_blog_posts_post_subtitle {
-        font-size: 1em;
-    }
-
     &.s_blog_post_list {
-        // Set sizes relative to the container font-size.
-        // (handle parents with, for example, '.small' or '.h1' classes)
-        .s_blog_posts_post_cover {
-            width: 3.5em;
-            height: 3.5em;
-            max: {width: 85px; height: 85px}
+        a {
+            color: inherit;
+        }
+        .s_blog_posts_post:hover {
+            background-color: $gray-100;
+        }
+        .s_blog_posts_post_avatar {
+            width: 24px;
+            height: 24px;
         }
 
-        .s_blog_posts_post_title  {
-            font-size: 1.25em;
+        .s_blog_posts_post_teaser {
+            max-height: 100px;
+            display: -webkit-box;
+            -webkit-line-clamp: 4;
+            -webkit-box-orient: vertical;
         }
     }
 
     &.s_blog_post_big_picture {
-        .s_blog_posts_post {
-            min-height: 150px;
-
-            figcaption {
-                position: relative;
-                justify-content: center;
-            }
-
-            .s_blog_posts_post_cover {
-                min-height: 100%;
-
-                .o_record_cover_container {
-                    top: 0;
-                }
-            }
-
-            .s_blog_posts_post_title  {
-                @include font-size($h3-font-size);
-                margin-bottom: 0.5em;
+        &.picture_size_small {
+            .s_blog_posts_post {
+                height: 220px;
             }
         }
+        .s_blog_posts_post {
+            height: 410px;
```

Comment:
Please let's not hardcode layout values; it's a responsive environment with (soon) responsive font-size. 
I'd suggest to check https://getbootstrap.com/docs/5.1/helpers/ratio/#aspect-ratios

Reply:
I tried to use these utility classes but there is an issue on larger screens when there is only 1 displayed element, the 1x1 ratio make the tiles look too big
<img width="1713" alt="Screenshot 2023-10-04 at 15 29 17" src="https://github.com/odoo/odoo/assets/110090660/e7934206-ca01-46f6-97a3-83fd11ea806f">